omod/src/main/java/org/openmrs/module/appointmentscheduling/web/controller/AppointmentListController.java

```diff
@@ -1,26 +1,13 @@
 package org.openmrs.module.appointmentscheduling.web.controller;

```

Comment:
*Lots* of formatting changes in this file, so hard to review... however, that being said, this is the old, old UI, so unless we are planning on modifying this functionality, I don't think this needs to change?